Formula/t/tcl-tk.rb

```diff
@@ -35,32 +35,36 @@ class TclTk < Formula
   conflicts_with "the_platinum_searcher", because: "both install `pt` binaries"
 
   resource "critcl" do
-    url "https://github.com/andreas-kupries/critcl/archive/refs/tags/3.2.tar.gz"
-    sha256 "20061944e28dda4ab2098b8f77682cab77973f8961f6fa60b95bcc09a546789e"
+    url "https://github.com/andreas-kupries/critcl/archive/refs/tags/3.3.1.tar.gz"
+    sha256 "d970a06ae1cdee7854ca1bc571e8b5fe7189788dc5a806bce67e24bbadbe7ae2"
   end
 
   resource "tcllib" do
-    url "https://downloads.sourceforge.net/project/tcllib/tcllib/1.21/tcllib-1.21.tar.xz"
-    sha256 "10c7749e30fdd6092251930e8a1aa289b193a3b7f1abf17fee1d4fa89814762f"
+    url "https://downloads.sourceforge.net/project/tcllib/tcllib/2.0/tcllib-2.0.tar.xz"
+    sha256 "642c2c679c9017ab6fded03324e4ce9b5f4292473b62520e82aacebb63c0ce20"
   end
 
+  # There is no tcltls release compatible with TCL 9 so using latest HEAD
+  # https://core.tcl-lang.org/tcltls/info/1505883e4a18b50e
+  # Ref: https://sourceforge.net/p/tcl/mailman/tcl-core/thread/eab3a8bf-b846-45ef-a80c-6bc94d6dfe91@elmicron.de/
   resource "tcltls" do
-    url "https://core.tcl-lang.org/tcltls/uv/tcltls-1.7.22.tar.gz"
-    sha256 "e84e2b7a275ec82c4aaa9d1b1f9786dbe4358c815e917539ffe7f667ff4bc3b4"
+    url "https://core.tcl-lang.org/tcltls/tarball/1505883e4a/tcltls-1505883e4a.tar.gz"
+    sha256 "a57d7b6b3710e6966387f0b2269e6a014b7b8b2db736e44d982af5318adeefba"
   end
```

Comment:
Added these, but may need to decide if we should just not ship them until an official release. Same for `itk`.